### ISSUE TITLE
[Legitimate Site Blocked] pulsechain.com

### ISSUE THREAD

**Issue by stephanjoseph** [MetaMask/eth-phishing-detect#6808]
https://pulsechain.com/

This site is not spam or phishing please unblock it.

**Comment by devenblake** [comment]
Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/6771.
